apps/condo/domains/ticket/schema/TicketComment.js

```diff
@@ -73,15 +75,54 @@ const TicketComment = new GQLListSchema('TicketComment', {
             schemaDoc: 'Plain text content',
             type: Text,
         },
-
     },
-    plugins: [uuided(), versioned(), tracked(), softDeleted(), historical()],
     hooks: {
+        resolveInput: async ({ operation, listKey, context, resolvedData, existingItem }) => {
+            const user = get(context, ['req', 'user'])
+            const userType = get(user, 'type')
+            const commentType = get(resolvedData, 'type')
+
+            if (operation === 'create' && commentType === RESIDENT_COMMENT_TYPE) {
+                const ticketId = get(resolvedData, 'ticket')
+                const dv = get(resolvedData, 'dv')
+                const sender = get(resolvedData, 'sender')
+                const now = new Date().toISOString()
+
+                if (userType === RESIDENT) {
+                    await Ticket.update(context, ticketId, {
```

Comment:
this will actually run `before` comment is created, so if comment creation fails, this update will kinda break data  integrity